app/src/interfaces/input-rich-text-html/useInlineCode.ts

```diff
@@ -0,0 +1,62 @@
+import { Ref } from 'vue';
+
+type InlineCodeButton = {
+	icon: string;
+	tooltip: string;
+	onAction: () => void;
+	onSetup: (api: { setActive: (isActive: boolean) => void }) => () => void;
+};
+
+type UsableInlineCode = {
+	inlineCodeButton: InlineCodeButton;
+};
+
+function isInlineCode(node: Node): boolean {
+	// check if node is a code node, or if it has a code node as a child
+	return node.nodeName === 'CODE' || (node as Element).querySelector('code') !== null;
+}
+
+export default function useInlineCode(editor: Ref<any>): UsableInlineCode {
+	let keydownHandler: ((event: KeyboardEvent) => void) | null = null;
+
+	const inlineCodeButton: InlineCodeButton = {
+		tooltip: 'InlineCode',
```

Comment:
Let’s use the i18n message here, otherwise the tooltip will display "InlineCode".

```suggestion
		tooltip: i18n.global.t('wysiwyg_options.codeblock'),
```

Reply:
Be sure to add the import